src/models/v2/schema.ts

```diff
@@ -8,8 +8,10 @@ import type { ExternalDocumentationInterface } from '../external-docs';
 import type { SchemaInterface } from '../schema';
 
 import type { v2 } from '../../spec-types';
+import { getDefaultSchemaFormat } from '../../schema-parser';
+import { DetailedAsyncAPI } from 'types';
 
-export class Schema extends BaseModel<v2.AsyncAPISchemaObject, { id?: string, parent?: Schema }> implements SchemaInterface {
+export class Schema extends BaseModel<v2.AsyncAPISchemaObject, { id?: string, asyncapi?: DetailedAsyncAPI, parent?: Schema, schemaFormat?: string }> implements SchemaInterface {
```

Comment:
`DetailedAsyncAPI` you have in `_meta` - it has all models https://github.com/asyncapi/parser-js/blob/master/src/models/base.ts#L12

Reply:
fixed